AirCasting/SessionViews/ABMeasurementsView.swift

```diff
@@ -67,7 +67,7 @@ struct _ABMeasurementsView: View {
                             Spacer()
                         }
                     }
-                }
+                }.padding(.bottom, 10)
```

Comment:
The task is "dashboard > increase space between parameter values & graph/map buttons" so is adding additional padding here not an overkill? I mean this will add space between ABMEasurementsView and all others while as far as I understand we should add it only fr fixed sessions between parameters and buttons?

Reply:
So maybe you could add a padding in SessionCardView in this method? What do you think? 
```
func displayButtons(thresholds: [SensorThreshold]) -> some View {
        HStack() {
            if sessionCartViewModel.isFollowing && session.type == .fixed {
                unFollowButton
            } else if session.type == .fixed {
                followButton
            }
            Spacer()
            !session.isIndoor ? mapButton.padding(.trailing, 10) : nil
            graphButton
        }
        .buttonStyle(GrayButtonStyle())
    }

```

Reply:
Adding comments to my comments XD But I see that dormant sessions also have parameters + buttons so we should take them into account, too. :D